en-US/privacy/index.lang

```diff
@@ -37,19 +57,36 @@ Mozilla Websites, Communications &amp; Cookies
 Firefox Browser
 
 
-;Firefox + Cliqz
-Firefox + Cliqz
+;Firefox for Fire TV
+Firefox for Fire TV
 
 
-;Firefox Cloud Services
-Firefox Cloud Services
+## TAG: updates-052018
+;Transparency Report
+Transparency Report
 
 
+## TAG: updates-052018
+;As an open source project, transparency and openness are an essential part of Mozilla’s founding principles. Our codebases are open and auditable. Our development work is open. Our bi-annual <a href="%(report)s">Transparency Report</a> also demonstrates our commitment to these principles.
+As an open source project, transparency and openness are an essential part of Mozilla’s founding principles. Our codebases are open and auditable. Our development work is open. Our bi-annual <a href="%(report)s">Transparency Report</a> also demonstrates our commitment to these principles.
 
+
+## TAG: updates-052018
+;To review and comment on proposed changes to our privacy policies <a href="%(group)s"> subscribe to Mozilla’s governance group</a>.
```

Comment:
This is not the string in the PR. As said, strings should be extracted via script, not manually.

Reply:
I am really confused with the changes in this PR. When I extract the strings, I get the upper case Governance Group. But the tagged strings suggest lower case.

Reply:
Besides a different capitalization for Governance Group, there's a comma before the link in the string behind `updates-052018`
https://github.com/mozilla/bedrock/pull/5850/files#diff-a8503aba570309aa9167f580d02641edR137